cmd/root.go

```diff
@@ -33,7 +33,7 @@ var rootCmd = &cobra.Command{
 		}
 
 		// Create http server
-		m, err := http.NewServer(uint16(viper.GetUint("server-port")))
+		m, err := http.NewServer(string(viper.GetString("server-addr")), uint16(viper.GetUint("server-port")))
```

Comment:
```suggestion
		m, err := http.NewServer(viper.GetString("server-addr"), uint16(viper.GetUint("server-port")))
```
This is sort of a nitpick but the `string()` is a redundant type conversion. I only do it for `server-port` because the Viper framework doesn't have a `viper.GetUint16()` but `viper.GetString()` returns a normal string.